components/terrain/chunkmanager.cpp

```diff
@@ -220,8 +220,14 @@ osg::ref_ptr<osg::Node> ChunkManager::createChunk(float chunkSize, const osg::Ve
         layer.mDiffuseMap = compositeMap->mTexture;
         layer.mParallax = false;
         layer.mSpecular = false;
-        geometry->setPasses(::Terrain::createPasses(mSceneManager->getForceShaders() || !mSceneManager->getClampLighting(), mSceneManager->getForcePerPixelLighting(),
-                                                    mSceneManager->getClampLighting(), &mSceneManager->getShaderManager(), std::vector<TextureLayer>(1, layer), std::vector<osg::ref_ptr<osg::Texture2D> >(), 1.f, 1.f));
+        geometry->setPasses(::Terrain::createPasses(mSceneManager->getForceShaders(),
+                                                    mSceneManager->getForcePerPixelLighting(),
+                                                    mSceneManager->getClampLighting(),
+                                                    mSceneManager->getAccurateFog(),
+                                                    &mSceneManager->getShaderManager(),
+                                                    std::vector<TextureLayer>(1, layer),
+                                                    std::vector<osg::ref_ptr<osg::Texture2D>>(),
+                                                    1.f, 1.f);
```

Comment:
The extra line breaks make it harder to see what was changed here, but I see that `mSceneManager->getForceShaders() || !mSceneManager->getClampLighting()` got changed to `mSceneManager->getForceShaders()` which I suspect isn't right.

Reply:
Good catch!

Just to be clear, I'm only reformatting due to some lines in excess of 200 columns.

Reply:
Some of us have wide monitors, so lines of 200 characters aren't always unreadable, and as word wrap is a thing, it's not the biggest concern even if there are. Sometimes, splitting is nice, though, but splitting for every individual argument can be excessive.

Reply:
Word wrap doesn't normally produce good results. It helps with horizontal scrolling, but messes up formatting and layout cleanliness. Personally, I avoid lines over 100 wide, and use indented line contiunations. e.g.
```
geometry->setPasses(::Terrain::createPasses(mSceneManager->getForceShaders(),
    mSceneManager->getForcePerPixelLighting(), mSceneManager->getClampLighting(),
    mSceneManager->getAccurateFog(), &mSceneManager->getShaderManager(),
    std::vector<TextureLayer>(1, layer), std::vector<osg::ref_ptr<osg::Texture2D>>(),
    1.f, 1.f
));
```
Makes it easy to see without horizontal scrolling or excessive bunching up on the right side, and can easily find where a call's parameter list ends.

Reply:
In an IDE, I have always-visible project filesystem structure and file's functions listed, taking close to half of the screen. A terminal takes about a fifth of the screen to aid multitasking. Given a 4K display, these environments have about ~140 columns each for text editing.

In any case I reformatted only the pieces of code that were touched for other reasons.